src/expressions/functions/string.test.ts

```diff
@@ -131,4 +131,12 @@ describe("String Functions", () => {
             param1: "NFC",
         });
     });
+
+    test("trim with trim expression", () => {
```

Comment:
I believe the simple case is not tested such as:
`trim('   hello   ')`

Reply:
That is already tested for the normal trim case, along with other functions of 1 parameter